Fix build on openSUSE

hillwoodroc opened the pull request.

BLumia commented on src/apps/dde-desktop/CMakeLists.txt:
```diff
@@ -2,6 +2,9 @@ cmake_minimum_required(VERSION 3.10)
 
 project(dde-desktop)
 
+set(CMAKE_C_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
```
```suggestion
set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
```

下同。这全错了= =

hillwoodroc replied:
汗，复制粘贴一把梭没看清楚。这回都改了。

hillwoodroc committed:
fix: add missing library link

Add missing library link

Log: Should link these libraries for some distributions

hillwoodroc committed:
fix: fix versions of libraries

Fix versions of libraries

Log: Versions of libraries are not identical.

BLumia commented:
/merge
